lib/vicadmin/validate.go

```diff
@@ -180,7 +180,7 @@ func NewValidator(ctx context.Context, vch *config.VirtualContainerHostConfigSpe
 	nwErrors := []error{}
 
 	for _, host := range hosts {
-		conn, err := net.Dial("tcp", host)
+		conn, err := net.DialTimeout("tcp", host, time.Minute/2)
```

Comment:
We discussed this on slack @caglar10ur and figured that Minute / 2 will be optimized to a bit shift so it doesn't matter